### ISSUE TITLE
PSNR values do not cross bicubic PSNR

### ISSUE THREAD

**Issue by burhanmudassar** [Jongchan/tensorflow-vdsr#7]
Hi,

I am using aug_train.m on the 91 image dataset to generate the image patches for training. I have set the base LR as 0.1 and the base NORM is set to 0.2. My issue is that the VDSR PSNR never seems to cross the bicubic PSNR values.

I have tried the following combinations

1. LR = 0.1 and decrease LR by a factor of 10 after every 20 epochs
2. LR = 0.1 and decrease LR by a factor of 10 after every 20 epochs and increase norm by a factor of 10 as well
In both cases I have reached about 60 epochs and a learning rate of 0.0001 and the PSNR on Set 5 never seems to grow higher than the Bicubic PSNR

**Comment by Nord786** [comment]
I have the same problem with this repository. I found work code in https://github.com/Nord786/SRCNN-Tensorflow (it's my fork) - it doen't calc PSNR, but it makes images on test and you can see quality by eye - it better then bicubic

**Comment by Jongchan** [comment]
I have noticed some problems with the current code. I think some of the hyperparameters are not up-to-date. I'm currently testing the local setups, will get back to you in a few days.

**Comment by Jongchan** [comment]
@burhanmudassar @Nord786 
I've updated the code. 
 AdamOptimizer stabilizes the training very much. This is only the first trial using AdamOptimizer, but it exceeds the best previous training result, after around 20~30 epochs.
